resource-validator/src/test/scala/com/broadinstitute/dsp/resourceValidator/DbReaderGetDeleteDiskCandidatesSpec.scala

```diff
@@ -0,0 +1,28 @@
+package com.broadinstitute.dsp.resourceValidator
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import com.broadinstitute.dsp.DbTestHelper.{insertDisk, isolatedDbTest, yoloTransactor}
+import com.broadinstitute.dsp.Generators.genDisk
+import com.broadinstitute.dsp._
+import doobie.Transactor
+import doobie.scalatest.IOChecker
+import org.scalatest.flatspec.AnyFlatSpec
+
+class DbReaderGetDeleteDiskCandidatesSpec extends AnyFlatSpec with CronJobsTestSuite with IOChecker {
+  implicit val config: DatabaseConfig = ConfigSpec.config.database
+  implicit val transactor: Transactor[IO] = yoloTransactor
+
+  it should "return disks that have been deleted without destroyedDate for over an hour in the Leo DB" taggedAs DbTest in {
```

Comment:
the test is mostly working...I still get some weird error locally which I think is irrelevant to the test